### ISSUE TITLE
Windows10 - Pinned to taskbar Icon gets unpinned on clicking

### ISSUE THREAD

**Issue by kalvelukas** [johannesjo/super-productivity#1004]
Environment
* Version used: 6.5.2
* Operating System and version: Windows 10 Pro,Version 20H2

### Expected Behavior
Pinned App Symbol in Taskbar should open the program and stay pinned

### Current Behavior
icon gets unpinned and vanishes from taskbar on termination of the app

### Steps to Reproduce (for bugs)
1. start SuperProductivity
2. pin to taskbar
3. close SuperProductivity
4. start again, Icon gets unpinned

**Comment by johannesjo** [comment]
Hi there! This sounds more like a windows question to me than one specific to the app :) But maybe somebody knows a solution to the problem?

**Comment by kalvelukas** [comment]
Hey hey, in previous versions of SuperProductivity this problem didn't occur. Only after I upgraded to the newer version.
But yes, help is appreciated. 👍

**Comment by ComputerCarl** [comment]
I am using Windows 10 Pro v 1903 (build 18362.267) and SP 6.5.2 and I do experience this issue.
The problem occurs if the running app is pinned the the taskbar, but not if a shortcut to the app is pinned.

If I click the shortcut after exiting SP, I get an error the executable is missing.
![image](https://user-images.githubusercontent.com/29293242/114136421-3d928780-993d-11eb-8099-3f112a5632bf.png)

However, if I use a shortcut to pin to taskbar, running the app will result in a second icon:
![image](https://user-images.githubusercontent.com/29293242/114136647-a0841e80-993d-11eb-836c-385f9cb66cc5.png)

**Comment by kalvelukas** [comment]
That's the exact same issue i am experiencing.

**Comment by kalvelukas** [comment]
I reinstalled the program via the setup.exe from the release tab.
the problem went away for now.
is the installer in the ms-store not the same?

**Comment by johannesjo** [comment]
@kalvelukas yes they are unfortunately. I don't know all the details to be honest, but the format is different and there are some things added to the executable when uploading it to the store.
